jme3-networking/src/main/java/com/jme3/network/message/ChannelInfoMessage.java

```diff
@@ -69,6 +70,6 @@ public int[] getPorts() {
     
     @Override
     public String toString() {
-        return "ChannelInfoMessage[" + id + ", " + Arrays.asList(ports) + "]";
+        return "ChannelInfoMessage[" + id + ", " + Collections.singletonList(ports) + "]";
```

Comment:
Another case where I think the authors original intent was to output the contents of the array and not just wrap the garbage string in brackets.

Reply:
Arrays.toString used instead